pootle_pytest/factories.py

```diff
@@ -41,3 +45,112 @@ class AgreementFactory(factory.django.DjangoModelFactory):
 
     class Meta(object):
         model = 'staticpages.Agreement'
+
+
+class DirectoryFactory(factory.django.DjangoModelFactory):
+
+    @factory.lazy_attribute
+    def pootle_path(self):
+        if self.parent is None:
+            return "/"
+        return (
+            "%s/%s"
+            % (self.parent.pootle_path.rstrip("/"),
+               self.name))
+
+    class Meta:
```

Comment:
This commit 5d0f7b77da62342ef2c2949da8de6a7090c12fd0 is about it, picked up here https://www.quantifiedcode.com/app/issue_class/86d4d89288cc4da9a6c1ad122f00771c